config/environments/development.rb

```diff
@@ -38,4 +38,16 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  Paperclip.options[:command_path] = "/opt/local/bin/convert"
+
+  config.paperclip_defaults = {
+    storage: :s3,
+    s3_host_name: "s3-us-west-1.amazonaws.com",
+    s3_credentials: { 
+      bucket: "cardsharper",
+      access_key_id: ENV['AWS_ACCESS_KEY_ID'],
+      secret_access_key: ENV['AWS_SECRET_ACCESS_KEY']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.